src/diffusers/models/autoencoders/autoencoder_kl_cogvideox.py

```diff
@@ -999,6 +999,7 @@ def __init__(
         # setting it to anything other than 2 would give poor results because the VAE hasn't been trained to be adaptive with different
         # number of temporal frames.
         self.num_latent_frames_batch_size = 2
+        self.num_sample_frames_batch_size = 8
```

Comment:
Can this be configurable? Why change to a different hardcoded value?

Reply:
This can be configurable to multiples of `temporal_compression_ratio` which is `4`. We can configure this using `vae.num_sample_frames_batch_size = 16` for the moment, and I think it's best to do it that way. 8, here, corresponds to 1 second of video because Cog was trained on 8 FPS videos. I would leave the configuration option out of `enable_tiling` and keep it for power users who have looked through the code and understand its usage tbh